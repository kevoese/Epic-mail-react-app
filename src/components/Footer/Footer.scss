.footer {
	display: grid;
	grid-template-columns: 20% 20%;
	position: relative;
	left: 7%;
	width: 80%;
	color: rgba(255, 255, 255, 0.8);
	font-family: myfont;
	font-size: 0.9em;
	margin-bottom: 3em;

	div {
		white-space: nowrap;
		animation: opace 0.4s 1.5s linear both;
		position: relative;

		div::after {
			font-size: 2.5em;
			color: white;
			left: 0;
			position: absolute;
			transform: translateY(-120%);
		}

		.allhrs {
			&::after {
				content: '24';
				font-family: serif;
			}
		}

		.secure {
			color: blueviolet;
			&::after {
        content: '\f3ed';
        transform: translateY(-135%);
			}
		}
	}
}
