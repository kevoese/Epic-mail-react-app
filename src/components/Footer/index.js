import React from 'react';
import './Footer';

const Footer = () => {
  return (
    <div className="footer">
      <div>
        <div className="allhrs"></div> <span>HOURS SERVICES</span>
      </div>
      <div>
        <div className="secure icon"></div>
        <span>SECURE INFORMATION</span>
      </div>
    </div>
  );
};

export default Footer;
