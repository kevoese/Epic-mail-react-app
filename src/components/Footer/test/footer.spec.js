/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import 'jsdom-global/register';
import Footer from '@components/Footer';

describe('Footer component', () => {
  it('should render the Footer component', () => {
    const component = shallow(<Footer />);
    expect(component).toMatchSnapshot();
  });
});
