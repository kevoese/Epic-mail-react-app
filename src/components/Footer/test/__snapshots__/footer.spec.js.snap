// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render the Footer component 1`] = `
<div
  className="footer"
>
  <div>
    <div
      className="allhrs"
    />
     
    <span>
      HOURS SERVICES
    </span>
  </div>
  <div>
    <div
      className="secure icon"
    />
    <span>
      SECURE INFORMATION
    </span>
  </div>
</div>
`;
